requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
-pytest
-pytest-cov
-pytest-django
+pytest==2.8.5
```

Comment:
Yes, standardizing on how the others are setup now and will get to the latest version everywhere for v1.0